myapp/spec/system/task_spec.rb

```diff
@@ -0,0 +1,66 @@
+require 'rails_helper'
+
+RSpec.describe Task, type: :system do
+  kaminari_data_per_page = 5
```

Comment:
`Task.default_per_page`で、モデル側で設定したページ数を取得することができるようです。
また、RSpec内でテスト対象のクラスを記載する時（今回でいうとTask）、`described_class`というメソッドが用意されています。
最終的に以下のような形で書けると思います！
```suggestion
  kaminari_data_per_page = described_class.default_per_page
```

Reply:
上記ご教示頂き有難うございます！
このように書けるのですね、、勉強になります！

頂いた内容で修正しました。
Rspecでの書き方含め、勉強しておきます。